buildkite/bazelci.py

```diff
@@ -44,170 +44,245 @@
 
 
 DOWNSTREAM_PROJECTS = {
-    # TODO(https://github.com/googlesamples/android-testing/issues/213): Reenable once fixed
-    # "Android Testing": {
-    #     "git_repository": "https://github.com/googlesamples/android-testing.git",
-    #     "http_config": "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/pipelines/android-testing-postsubmit.yml"
-    # },
-    # TODO(bazel#6288): enable once remote execution is green
-    # "Bazel Remote Execution": {
-    #     "git_repository": "https://github.com/bazelbuild/bazel.git",
-    #     "http_config": "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/pipelines/bazel-remote-execution-postsubmit.yml"
-    # },
-     # TODO(https://github.com/bazelbuild/BUILD_file_generator/issues/39): reenable once fixed
-#     "BUILD_file_generator": {
-#         "git_repository": "https://github.com/bazelbuild/BUILD_file_generator.git",
-#         "http_config": "https://raw.githubusercontent.com/bazelbuild/BUILD_file_generator/master/.bazelci/presubmit.yml"
-#     },
+    "Android Testing": {
+        "git_repository": "https://github.com/googlesamples/android-testing.git",
+        "http_config": "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/pipelines/android-testing-postsubmit.yml",
+        "pipeline_slug": "android-testing",
+        # TODO(https://github.com/googlesamples/android-testing/issues/213): Reenable once fixed
+        "downstream_enabled": False
```

Comment:
We could replace "downstream_enabled" with a string field "disabled_reason". This would give us several benefits:
1. We don't have to keep the comments around.
2. We would enforce having a reason for disabling downstream projects.
3. We could even print that reason as part of the pipeline so that someone wondering "why doesn't this pipeline run" can understand everything by looking at the logs. I'm not particularly happy that people have to look at this script to understand what their pipeline build is doing.

Reply:
Thanks for this suggestion! PTAL~